cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/support/Beans.java

```diff
@@ -0,0 +1,95 @@
+package org.apereo.cas.configuration.support;
+
+import com.zaxxer.hikari.HikariDataSource;
+import org.apache.commons.lang3.StringUtils;
+import org.apereo.cas.configuration.model.support.jpa.AbstractJpaProperties;
+import org.apereo.cas.configuration.model.support.jpa.DatabaseProperties;
+import org.apereo.cas.configuration.model.support.jpa.JpaConfigDataHolder;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+
+import java.util.Properties;
+
+/**
+ * A re-usable collection of utility methods for object instantiations and configurations used cross various
+ * <code>@Bean</code> creation methods throughout CAS server.
+ *
+ * @author Dmitriy Kopylenko
+ * @since 5.0.0
+ */
+public class Beans {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://www.codacy.com/app/mmoayyed/cas/file/2770326251/issues/source?bid=3256115&fileBranchId=3357337#l20)